backend/mark_marker/src/main/scala/mark_marker/trainer/DatabaseConnector.scala

```diff
@@ -0,0 +1,96 @@
+package mark_marker.trainer
+
+import java.io._
+import java.sql.DriverManager
+import java.sql.Connection
+
+import edu.stanford.nlp.classify.{Classifier, ColumnDataClassifier}
+
+import scala.io.Source
+
```

Comment:
could reuse the Java database connector here as well? Should be easy to import it from the infrastructure